File: C/Esercizi/0_Eserciziario/es14.c
/********************************************
LIVELLO: Facile
Scuola: Liceo, Secondo Anno
Conoscenza: Scanf, Printf, IF
*********************************************/

/*
14. Scrivere un algoritmo che, letti in input due numeri interi, verifichi se il primo è multiplo del secondo.
*/

#include <stdio.h>

int main(){ 
	int a;
  int b;
  printf ("inserisci due valori ");
  scanf ("%d %d", &a, &b);
	
	// se non da resto allora il primo è multiplo
  if (a%b==0){
    printf ("i due numeri sono multipi");
  }

	// in tutti gli altri casi...
  else {
    printf ("i due numeri non sono multipli");
  }
  
	return 0;
}

File: Java/Esercizi/0_Eserciziario/es14.java
import java.util.Scanner;

// Scrivere un algoritmo che, letti in input due numeri interi, verifichi se il primo è multiplo del secondo.

class es14 {

	public static void main(String[] args) {
		
		// creo un nuovo scanner per chiedere un numero intero
		Scanner tastiera = new Scanner(System.in);

		System.out.println("Inserisci il primo numero: ");
    int primonum = tastiera.nextInt();
		
		System.out.println("Inserisci il secondo numero: ");
    int secondonum = tastiera.nextInt();

    if((primonum % secondonum) == 0){
      System.out.println("primo numero e' multiplo del secondo");
    }
		
    else{
      System.out.println("il primo numero non e' multiplo del secondo");
    }

	}

}

